addons/im_livechat/static/src/embed/core/disabled_features.js

```diff
@@ -1,17 +1,17 @@
 /* @odoo-module */
 
-import { Composer } from "@mail/core/common/composer";
-import { Store } from "@mail/core/common/store_service";
+import { messageActionsRegistry } from "@mail/core/common/message_actions";
 import { threadActionsRegistry } from "@mail/core/common/thread_actions";
 import { Thread } from "@mail/core/common/thread_model";
 import { ThreadService } from "@mail/core/common/thread_service";
 
 import { patch } from "@web/core/utils/patch";
 import { SESSION_STATE } from "./livechat_service";
 
-patch(Composer.prototype, {
-    get allowUpload() {
-        return false;
+const downloadFilesAction = messageActionsRegistry.get("download_files");
+patch(downloadFilesAction, {
+    condition(component) {
+        return component.message.originThread.type !== "livechat" && super.condition(component);
```

Comment:
Why would download_files be prevented? If there are files it should be available?

Reply:
I'd rather enable it in a follow up PR:
- This one is already big enough
- I need to adapt the message action declaration in order to allow route/param override which would add more diff since the request is done by XHR.
- It's just a bonus, not a primordial feature.